app/internal/languages.py

```diff
@@ -0,0 +1,85 @@
+import glob
+import json
+import pathlib
+from typing import Dict, Union
+
+from app import config
+
+LANGUAGE_FILES_PATH = "app/languages/*.json"
+LANGUAGE_FILES_PATH_TEST = "../app/languages/*.json"
+
+translations_dict = {}
+
+
+def get_translations_dict() -> Dict[str, Union[str, Dict[str, str]]]:
```

Comment:
Prefer `get_translation_words` to give more information about the content, and leave out the Hungarian notation